core/src/main/java/org/infinispan/topology/ClusterCacheStatus.java

```diff
@@ -139,153 +140,143 @@ public AvailabilityMode getAvailabilityMode() {
    }
 
    @Override
-   public void updateAvailabilityMode(List<Address> actualMembers, AvailabilityMode newAvailabilityMode,
+   public synchronized void updateAvailabilityMode(List<Address> actualMembers, AvailabilityMode newAvailabilityMode,
```

Comment:
I really prefer concise code, when one keyword can replace two extra lines and indentation level.